### DISCUSSION TITLE
Installing various applications in one command

### DISCUSSION THREAD

**Discussion by RafaelLinux** [openSUSE/opi#154] [Ideas]
After installing and upgrading/update, I usually add various applications, like puggletag, anydesk, codecs, msedge, chrome, copyq .... and alwyas use "-n" options. Could you let user to add more than one application to install in only one command??

Thank you

**Comment by nobkd** [comment]
I made a PR that should allow this: #157

---

The command *then* could look like this:
```
sudo opi -nm package1 package2
```
This should also allows plugins (they always run first, regardless of the package order).

**Reply by nobkd** [reply]
This was released with [v3.6.0](https://github.com/openSUSE/opi/releases/tag/v3.6.0) and should be available soon.

**Comment by RafaelLinux** [comment]
Fantastic!!!! Fast and great way to implement it.

Thank you!!
